layouts/home.vue

```diff
@@ -1,9 +1,7 @@
 <template>
   <div>
-    <div>
-      <Navbar />
```

Comment:
should the components be named like CamelCase or how? Why "Navbar" instead of "navbar"?

Reply:
Yes, kebab-case for folders under "components" folder and PascalCase for components